spec/features/api_particulier/prolong_token_build_spec.rb

```diff
@@ -59,7 +97,7 @@
         expect(token.reload.exp).to eq(18.months.from_now.to_i)
       end
 
-      it 'is on happy path and it requires update' do
+      it 'requires update' do
```

Comment:
Je changerais le titre ici parce qu'on teste un user path plutôt que le fait que ça requiert une update

```suggestion
      it 'follows update path' do
```